06ubuntu_apache/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ubuntu:latest
+
+MAINTAINER Eugene Naryshkin <aurcame@gmail.com>
+
+RUN apt-get update && \ 
+apt-get install apache2 -y && \ 
+service apache2 start
```

Comment:
Command 'service apache2 start' is not necessary here (in the 'docker build' commands chain) as service will be started at the container start stage.